### ISSUE TITLE
Support adding vhosts via command

### ISSUE THREAD

**Issue by mikebell** [opdavies/drupal-vm-config-generator#17]
It'd be nice to be able to add apache/nginx vhosts using this command once the config has been generated.

**Comment by opdavies** [comment]
Yes, there are definitely some potential applications for it that could be added, as well as generating the initial configuration.

This could be one of them. :smile:
